src/System.Private.ServiceModel/tests/Scenarios/SelfHostWcfService/Program.cs

```diff
@@ -10,12 +10,18 @@ internal class Program
     {
         private static void Main(string[] args)
         {
-            using (ServiceHost host = new ServiceHost(
-                typeof(WcfService)))
+            using (ServiceHost wcfservicehost = new ServiceHost(
+              typeof(WcfService)))
             {
-                host.Open();
-                Console.WriteLine("Service is ready!");
-                Console.ReadKey();
+                wcfservicehost.Open();
+                Console.WriteLine("WcfService is ready");
+
+                using (ServiceHost userNameService = new ServiceHost(typeof(WcfUserNameService)))
+                {
+                    userNameService.Open();
+                    Console.WriteLine("Wcf user name service is ready");
+                    Console.ReadLine();
```

Comment:
Nit: it would be useful to Console.WriteLine( "Press ENTER to close services...") if this window is meant to be visible.
This is a good example of where Bridge would probably be used to launch these services independently on demand.